packages/flutter/lib/src/animation/curves.dart

```diff
@@ -97,6 +108,9 @@ class Interval extends Curve {
 
 /// A curve that is 0.0 until it hits the threshold, then it jumps to 1.0.
 class Step extends Curve {
+  /// Creates a step cruve.
+  ///
+  /// The [threshold] argument must not be null.
   const Step(this.threshold);
 
   /// The value before which the curve is 0.0 and after which the curve is 1.0.
```

Comment:
presumably where the comment below says "step" it really means "`t=threshold`" or some such.